cv/qc-cv.obo

```diff
@@ -417,6 +417,1146 @@ is_a: QC:4000024 ! MS2 metric
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000009 ! ID based
+is_a: QC:4000006 ! table
+has_column: QC:4000081 ! Outliers below
+has_column: QC:4000082 ! Outliers above
+
+[Term]
+id: QC:40000
+name: Missed cleavages in identified peptides Q1, Q2, Q3
+def: "From the distribution of missed cleavages in identified peptides the quartiles Q1, Q2, Q3 value" [PSI:QC]
+comment: The missed cleavage distribution can aid the interpretation of digestion efficacy.
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000004 ! n-tuple
+has_units: STATO:0000167 ! first quartile, STATO:0000574 ! center value, STATO:0000170 ! third quartile
+
+[Term]
+id: QC:40000
+name: Missed cleavage mean
+def: "From the distribution of missed cleavages in identified peptides the mean" [PSI:QC]
+comment: The missed cleavage distribution can aid the interpretation of digestion efficacy.
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000003 ! single value
+has_units: STATO:0000401 ! sample mean
+synonym: "MS1-5B" NARROW []
+
+[Term]
+id: QC:40000
+name: Missed cleavage sigma
+def: "From the distribution of missed cleavages in identified peptides the sigma value" [PSI:QC]
+comment: The missed cleavage distribution can aid the interpretation of digestion efficacy.
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000003 ! single value
+has_units: STATO:0000237 ! standard deviation
+
+[Term]
+id: QC:40000
+name: Missed cleavage outliers (<Q1-1.5*IQR, >Q3+1.5*IQR)
+def: "From the distribution of missed cleavages in identified peptides, the list of outliers below of Q1-1.5IQR and above Q3+1.5IQR" [PSI:QC]
+comment: The missed cleavage distribution can aid the interpretation of digestion efficacy.
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000009 ! ID based
+is_a: QC:4000006 ! table
+has_column: QC:4000081 ! Outliers below
+has_column: QC:4000082 ! Outliers above
+
+[Term]
+id: QC:40000
+name: Identified peptide lengths Q1, Q2, Q3
+def: "From the distribution of identified peptide lengths the quartiles Q1, Q2, Q3 value" [PSI:QC]
+comment: The identified peptide lengths distribution can aid the interpretation of digestion efficacy.
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000004 ! n-tuple
+has_units: STATO:0000167 ! first quartile, STATO:0000574 ! center value, STATO:0000170 ! third quartile
+
+[Term]
+id: QC:40000
+name: Missed cleavage mean
+def: "From the distribution of identified peptide lengths the mean" [PSI:QC]
+comment: The identified peptide lengths distribution can aid the interpretation of digestion efficacy.
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000003 ! single value
+has_units: STATO:0000401 ! sample mean
+synonym: "MS1-5B" NARROW []
+
+[Term]
+id: QC:40000
+name: Missed cleavage sigma
```

Comment:
duplicate (see above)

.... maybe the OBO should be run through a script to detect plain duplicates?